src/AddItems.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+userToken = 'randomTokenOne'
```

Comment:
You'll need to define this - probably a `const`. This is causing the build to fail.

Reply:
Yea just doing that now!

Reply:
So fast!!

Reply:
THANK THE LORD! Build successful! 😭

Reply:
Awesome! Nice job, Karan!